tests/src/test/fdc3.raiseIntent.ts

```diff
@@ -1,32 +1,150 @@
 import { ResolveError } from "@finos/fdc3";
-
-const ExpectedErrorNotThrownError = "Expected error NoAppsFound not thrown";
+import { assert, expect } from "chai";
 
 export default () =>
-  describe("fdc3.raiseIntent", async () => {
-    it("Method is callable", async () => {
-      const context = {
-        type: "ThisContextDoesNotExist",
-        name: "Name",
-        id: {
-          ticker: "ticker",
-          ISIN: "US0378331005",
-          CUSIP: "037833100",
-          FIGI: "BBG000B9XRY4",
+  describe("fdc3.raiseIntent", () => {
+    before(async () => {
+      await window.fdc3.getOrCreateChannel("fdc3.raiseIntent");
+      await window.fdc3.joinChannel("fdc3.raiseIntent");
+    });
+
+    beforeEach(async () => {
```

Comment:
I think this would be better as an `afterEach` - cleaning up after the tests

Reply:
I prefer to always do things in the beforeEach of a test, that way you can be sure that apps have closed down before the test starts (rather than after, which might not happen if there are issues with other tests). 

However, I take your point about cleaning up after each test and will do both.